logging/overlays/nerc-ocp-obs/externalsecrets/openshift-logging-lokistack-gateway-bearer-token_patch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: openshift-logging-lokistack-gateway-bearer-token
+  namespace: openshift-logging
+spec:
+  dataFrom:
+    - extract:
+        # Command to extract the JSON pull secret: oc extract secret/pull-secret -n openshift-config --to=-
+        key: nerc/nerc-ocp-obs/lokistack-gateway-bearer-token
```

Comment:
Let's remove this ExternalSecret in `logging/overlays/nerc-ocp-obs` because the LokiStack will create it.